src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const priceADay = 40;
+  const totalCost = priceADay * days;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+
+  if (days >= 7) {
+    return totalCost - bigDiscount;
+  }
+
+  if (days >= 3) {
+    return totalCost - smallDiscount;
+  }
```

Comment:
The logic for applying discounts is correct. The function first checks if the number of days is 7 or more to apply the big discount, and then checks if it's 3 or more for the small discount. This ensures that the larger discount takes precedence when applicable.